### ISSUE TITLE
Expose image flag of oc cluster up to minishift start command.

### ISSUE THREAD

**Issue by praveenkumar** [minishift/minishift#2550]
As of now, we by default use images which are on the docker hub for minishift and there is no provision for the users if they want to use a different registry then docker hub to pull those images. 

A user can enable the experiment flag as of now and the use `extra-cluster-up` flags to accomplish it but this is still not directly facing for users.

Note: This is mostly for our downstream purpose when enterprise already has own registry which contains the required images so they don't need to fetch it from the internet or going through the proxy.

**Comment by LalatenduMohanty** [comment]
@praveenkumar I do not think that enterprises will build OpenShift images for `oc cluster up`. Or is it more useful for testing purposes?

**Comment by pjos** [comment]
This one would be very appreciated :-) Especially since i can't save "extra-cluster-up" content with `minishift config set` :-( https://github.com/minishift/minishift/issues/1664
